biojava-structure-gui/src/main/java/org/biojava/nbio/structure/align/gui/autosuggest/SCOPAutoSuggestProvider.java

```diff
@@ -179,7 +183,7 @@ public void clear() {
 	public void stop() {
 		stop.set(true);
 		if (DEBUG)
```

Comment:
See http://www.slf4j.org/faq.html#logging_performance

Personally, I think LOGGER.isDebugEnabled() is hateful.


> On 19 May 2016, at 15:38, Jose Manuel Duarte <notifications@github.com> wrote:
> 
> In biojava-structure-gui/src/main/java/org/biojava/nbio/structure/align/gui/autosuggest/SCOPAutoSuggestProvider.java:
> 
> > @@ -179,7 +183,7 @@ public void clear() {
> >  	public void stop() {
> >  		stop.set(true);
> >  		if (DEBUG)
> 
> Here we can either get rid of the DEBUG variable altogether or use LOGGER.isDebugEnabled(). The best solution is not having DEBUG at all and using the LOGGER.debug statements directly.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> 


-- 
The University of Edinburgh is a charitable body, registered in
Scotland, with registration number SC005336.